### PR TITLE
UIORGS-67 Add note field to contact person

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UIORGS-67
## Approach
Add field `Note` to `ContactPerson` and `EditContact` components
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js [hunk 1]
```diff
@@ -49,6 +49,12 @@ const ContactPersonDetails = (
           {categories}
         </KeyValue>
       </Col>
+
+      <Col xs={3}>
```
Comment (AndreiKandratsenka) [inline src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js]: Is it enough 3 column for notes?

Comment (AndreiShumski2) [reply src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js]: It's a good question. But we have mockup)
image

Comment (aliaksei-chumakou) [reply src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js]: so it won't use space at the right then. But Note value might be a large one, maybe it makes sense to put `xs={6}` here

**Comment by aliaksei-chumakou** [comment]
@AndreiShumski2 also please check organization details - contact people section. I think we need to show Note there also

File: src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js]: Ok, will update it

**Comment by AndreiShumski2** [comment]
Note field is also presented in Contact People accordion on Organization details pane

**Pull request merged**